blocks/embed/embed.js

```diff
@@ -76,20 +94,30 @@ const loadEmbed = (block, link, autoplay) => {
   ];
 
   const config = EMBEDS_CONFIG.find((e) => e.match.some((match) => link.includes(match)));
-  const url = new URL(link);
+  const url = link ? new URL(link) : '';
+  const blockConfig = readBlockConfig(block);
+
+  console.log('Block config' + JSON.stringify(blockConfig));
   if (config) {
     block.innerHTML = config.embed(url, autoplay);
     block.classList = `block embed embed-${config.match[0]}`;
+  } else if (block.classList.contains('brightcove')) {
+    const video = blockConfig.video ? blockConfig.video : '6331538546112';
```

Comment:
As you can see, I had to hardcode video id because config is not working for me.

Reply:
in the word document you have `videoId` and here you are accessing `blockConfig.video`. Try doing a `console.log(blockConfig)` to see what you have in the object.

Reply:
I do have a log statement committed. blockConfig is empty.

Reply:
This line removes everything from the DOM under the block: https://github.com/hlxsites/servicenow/blob/2f7749c46450fd4437d92702622907f3d7a09888/blocks/embed/embed.js#L121 . You should read the block config before that happens.